kornia/core/_backend_typing.pyi

```diff
@@ -0,0 +1,92 @@
+from typing import Any, Generator, List, Optional, Sequence, Tuple, Union, overload
```

Comment:
I am uncertain if this approach is a good idea.

Reply:
yeah, mostly because difficult to track and easy to get diverged

Reply:
Exactly, and easily something that doesn't have backward compatibility can be added and make it harder to trace the problem. I see that other possibilities might be better like:
1. maybe test typing only against the latest version of pytorch  
2. add `type: ignore` for specific cases that occurred in `make_sampler`.